lib/internal/constants.js

```diff
@@ -1,16 +1,136 @@
 'use strict';
 
-module.exports = {
-  // Alphabet chars.
-  CHAR_UPPERCASE_A: 65, /*A*/
-  CHAR_LOWERCASE_A: 97, /*a*/
-  CHAR_UPPERCASE_Z: 90, /*Z*/
-  CHAR_LOWERCASE_Z: 122, /*z*/
+const ASCII = {
+  NUL: 0,
+  SOH: 1,
+  STX: 2,
+  ETX: 3,
+  EOT: 4,
+  ENQ: 5,
+  ACK: 6,
+  BEL: 7,
+  BS: 8,
+  HT: 9,
+  LF: 10,
+  VT: 11,
+  FF: 12,
+  CR: 13,
+  SO: 14,
+  SI: 15,
+  DLE: 16,
+  DC1: 17,
+  DC2: 18,
+  DC3: 19,
+  DC4: 20,
+  NAK: 21,
+  SYN: 22,
+  ETB: 23,
+  CAN: 24,
+  EM: 25,
+  SUB: 26,
+  ESC: 27,
+  FS: 28,
+  GS: 29,
+  RS: 30,
+  US: 31,
+  SPACE: 32,
+  EXMARK: 33,
+  QUOTES: 34,
+  NUMSIGN: 35,
+  DOLLAR: 36,
+  PERCENT: 37,
+  AMPERSAND: 38,
+  APOSTROPHE: 39,
+  LPAREN: 40,
+  RPAREN: 41,
+  ASTERISK: 42,
+  PLUS: 43,
+  COMMA: 44,
+  HYPHEN: 45,
+  PERIOD: 46,
+  SLASH: 47,
+  DIGIT_0: 48,
+  DIGIT_1: 49,
+  DIGIT_2: 50,
+  DIGIT_3: 51,
+  DIGIT_4: 52,
+  DIGIT_5: 53,
+  DIGIT_6: 54,
+  DIGIT_7: 55,
+  DIGIT_8: 56,
+  DIGIT_9: 57,
+  COLON: 58,
+  SEMICOLON: 59,
+  LT: 60,
+  EQ: 61,
+  GT: 62,
+  QUES: 63,
+  AT: 64,
+  UPPER_A: 65,
+  UPPER_B: 66,
+  UPPER_C: 67,
+  UPPER_D: 68,
+  UPPER_E: 69,
+  UPPER_F: 70,
+  UPPER_G: 71,
+  UPPER_H: 72,
+  UPPER_I: 73,
+  UPPER_J: 74,
+  UPPER_K: 75,
+  UPPER_L: 76,
+  UPPER_M: 77,
+  UPPER_N: 78,
+  UPPER_O: 79,
+  UPPER_P: 80,
+  UPPER_Q: 81,
+  UPPER_R: 82,
+  UPPER_S: 83,
+  UPPER_T: 84,
+  UPPER_U: 85,
+  UPPER_V: 86,
+  UPPER_W: 87,
+  UPPER_X: 88,
+  UPPER_Y: 89,
+  UPPER_Z: 90,
+  LBRACK: 91,
+  BSLASH: 92,
```

Comment:
Can you spell out `BACKSLASH`, `NUMBER_SIGN`, `LEFT_BRACKET` etc?

Ideally we’d use something pretty close to the official Unicode identifiers, although those might be a bit too verbose sometimes.

Reply:
Sure!

Reply:
I personally also preferred the former more verbose naming with `LOWERCASE` and `UPPERCASE`.